modules/ROOT/pages/subqueries/subqueries-in-transactions.adoc

```diff
@@ -591,6 +602,189 @@ RETURN n.num, s.errorMessage;
 REPORT STATUS can only be used when specifying ON ERROR CONTINUE or ON ERROR BREAK
 ----
 
+[role=label--new-5.21]
+[[concurrent-transactions]]
+== Concurrent transactions
+
+By default, `CALL { ... } IN TRANSACTIONS` is single-threaded; one CPU core is used to sequentially execute batches.
+
+However, `CALL` subqueries can also execute batches in parallel by appending `IN [n] CONCURRENT TRANSACTIONS`, where `n` is a concurrency value used to set the maximum number of transactions that can be executed in parallel.
+This allows `CALL` subqueries to utilize multiple CPU cores simultaneously, which can significantly reduce the time required to execute a large, outer transaction.
+
+[NOTE]
+The concurrency value is optional.
+If not specified, a default value based on the amount of available CPU cores will be chosen. 
+
+.Load a CSV file in concurrent transactions
+====
+
+`CALL { ... } IN CONCURRENT TRANSACTIONS` is particularly suitable for importing data without dependencies.
+This example creates `Person` nodes from a unique `tmdbId` value assigned to each person row in the CSV file (444 in total) in 3 concurrent transactions.
+
+[source, cypher]
+----
+LOAD CSV WITH HEADERS FROM 'https://data.neo4j.com/importing-cypher/persons.csv' AS row
+CALL {
+  WITH row
+  CREATE (p:Person {tmdbId: row.person_tmdbId})
+  SET p.name = row.name, p.born = row.born
+} IN 3 CONCURRENT TRANSACTIONS OF 10 ROWS
+RETURN count(*) AS personNodes
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="m"]
+|===
+| personNodes
+| 444
+1+d| Rows: 1
+|===
+
+====
+
+[[non-determinism]]
+=== Concurrency and non-deterministic results
+
+`CALL { ... } IN TRANSACTIONS` uses _ordered_ semantics by default, where batches are committed in a sequential row-by-row order.
+For example, in `CALL { <I> } IN TRANSACTIONS`, any writes done in the execution of `<I~1~>` must be observed by `<I~2~>`, and so on.
+
+In contrast, `CALL { ... } IN CONCURRENT TRANSACTIONS` uses _concurrent_ semantics, where both the number of rows committed by a particular batch and the order of committed batches is undefined.
+That is,  in `CALL { <I> } IN CONCURRENT TRANSACTIONS`, writes committed in  execution of `<I~1~>` may not be observed by `<I~2~>`, and so on.
```

Comment:
I think maybe we can be even more clear that it may also be observed, i.e. use "may or may not be observed"